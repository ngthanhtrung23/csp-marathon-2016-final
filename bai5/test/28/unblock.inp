.11.2.
333.2.
004.25
674885
6799.A
BBB..A
