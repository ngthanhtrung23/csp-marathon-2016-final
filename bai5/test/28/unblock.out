28
1 L 1
5 U 1
9 R 1
A U 1
B R 3
4 D 2
6 D 1
7 D 1
8 L 3
4 U 2
9 L 1
2 D 2
B L 1
A D 1
5 D 1
3 R 3
4 U 2
0 R 2
8 R 2
7 U 3
8 L 2
B L 1
2 D 1
0 R 1
4 D 2
3 L 1
5 U 2
0 R 1

